developer_docs/Intallation.md

```diff
@@ -0,0 +1,36 @@
+# Setting up the development environment
+
+Follow the below steps to setup the development environment which you need before  contributing to the project
+
+## Prerequesites
+
+The main libraries used in the project are ReactJS,NextJS and NodeJS,NPM as the dependency management tool.
```

Comment:
@JithinKS97 We are using yarn for package management, right?

Reply:
I faced some issues with yarn when trying to install the fabric canvas library. So switched to npm.

Reply:
Okay

Reply:
@Anu-Asok Should we also consider adding instructions for setting up the back end dev environment as well in the doc?

Reply:
@JithinKS97 I’ll add the instructions for backend dev env setup in a separate doc once I’m done creating basic GraphQL API’s. Let’s not mix up things.